index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+  <style>
+    body {
+      display: block;
+      font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
+    }
+
+    #main {
+      background-color: #eee;
+      margin: 30px 10px;
+      border-radius: 10px;
+    }
+
+    #title {
+
+      font-size: 2.5rem;
+      text-align: center;
+      width: 20vw;
+      margin: 2px auto 20px;
+      padding: 10px;
+    }
+
+    #title+p {
+      text-align: center;
+      width: 20vw;
+      margin: 2px auto 20px;
+    }
+
+    #img-div {
+      background-color: white;
+      display: block;
+      width: 50vw;
+      margin: 2px auto 2px;
+      padding: 0px 430px;
+    }
+
+    #img-div img {
+      width: 1000px;
+      height: 600px;
+    }
+
+    #img-caption {
+      text-align: center;
+      padding: 10px;
+    }
+    #tribute-link h2{
+      text-align: center;
+      display: block;
+      width: 25vw;
+      padding: 20px 2px;
+      margin: 2px auto 2px;
+    }
+    #tribute-link ul{
+      display: block;
+      width: 25vw;
+      margin: 2px auto 2px;
+    }
+    #tribute-link ul li{
+      font-weight: bold;
+      margin: 27px 0px;
+    }
+    #tribute-link ul li span{
+      font-weight: lighter;
+    }
+    .quote{
+      font-style: italic;
+      padding: 5px;
+      display: block;
+      width: 25vw;
+      margin: 2px auto 2px;
+    }
+    .quotep{
+      font-style: italic;
+      padding: 10px;
+      text-align: right;
+      display: block;
+      width: 30vw;
+      margin: 2px auto 2px;
+    }
+    #footer{
+      color: rgb(54, 54, 54);
+      font-size: 1.2rem;
+      font-weight: bold;
+      text-align: center;
+      display: block;
+      width: 50vw;
+      padding: 20px;
+      margin: 2px auto 2px;
+    }
+
+
+  </style>
+</head>
+
+<body>
+  <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script> -->
+
+  <main id="main">
+    <h1 id="title">Alan Turing</h1>
+    <p>The inventor of the first computer</p>
+    <figure id="img-div">
+      <img src="https://www.denverpost.com/wp-content/uploads/2020/02/Code_Breaker_Memorabilia_51691.jpg">
```

Comment:
It is always a good idea to add an alt attribute to the images that are part of the content.